test-runner.js

```diff
@@ -1,11 +1,25 @@
 #!/usr/bin/env node
-var fs = require("fs");
-var path = require("path");
-var portastic = require("portastic");
-var child_process = require("child_process");
 
-var verbose = process.env.npm_config_loglevel === "verbose";
-var capabilitiesLocation = process.env.npm_config_capsLocation;
+const fs = require("fs");
+const path = require("path");
+const portastic = require("portastic");
+const child_process = require("child_process");
+const utils = require("./utils");
+let appium = "appium";
```

Comment:
nope
cause there is another check if we run tests on windows and it should be init again like 

if (process.platform === "win32") {
    appium = "appium.cmd";
    mocha = "mocha.cmd";
}